assets/js/romo/keypress.js

```diff
@@ -0,0 +1,35 @@
+$.fn.romoKeypress = function() {
+  return $.map(this, function(element) {
+    return new RomoKeypress(element);
+  });
+}
+
+var RomoKeypress = function(element) {
+  this.elem = $(element);
+  this.defaultTriggerOn = 'keydown';
```

Comment:
@jcredding I'm going to keep this as 'keydown' for now.  'keypress' is only triggered on character inputs (but not for modifiers).  I think it is better to detect all keys by default.